mrp_operating_unit/models/mrp.py

```diff
@@ -47,7 +47,9 @@ def _check_location_operating_unit(self):
     @api.onchange("operating_unit_id")
     def _onchange_operating_unit_id(self):
         """Change locations according to the warehouse of the operating unit"""
-        if not self.operating_unit_id:
+        if (not self.operating_unit_id
```

Comment:
Well done, by adding the stop condition with the other new onchange not to get an endless ping-pong